Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.4-alpine3.10
+
+COPY . .
+
+RUN ["apk", "--update", "add", "py-pip", "alpine-sdk"]
+
+RUN ["pip3", "install", "iteration_utilities", "pycryptodome"]
```

Comment:
alpine SDK meta-package may include make so this could be replaced with `make depends` - that way if the dependencies change the Dockerfile won't need updating